### PR TITLE
Improve glob summary text

### PR DESCRIPTION
**Pull request by ghogen**
## Summary

Improve text for readability and translation.

### CODE REVIEW

**Comment by ghogen** [comment]
@v-alje @gewarren @JasonCard  I think this is ready to be merged to address the GlobResult.xml issues we were discussing.

File: xml/Microsoft.Build.Evaluation/GlobResult.xml [hunk 1]
```diff
@@ -139,7 +140,9 @@
       </ReturnValue>
       <Docs>
         <summary>
-            A <see cref="T:Microsoft.Build.Globbing.IMSBuildGlob" /> representing the include globs. It also takes the excludes and relevant removes into consideration.
+            A <see cref="T:Microsoft.Build.Globbing.IMSBuildGlob" /> representing the globs found in an item `Include` attribute.
+            It also takes into consideration the strings from `Exclude` attributes that are present on that item, and strings from
```
Comment (gewarren) [inline xml/Microsoft.Build.Evaluation/GlobResult.xml:144]: I'm unclear what "takes into consideration" means. It's also not clear what "it" refers to.

**Comment by gewarren** [comment]
@ghogen Did you want to take a look at this feedback?

**Comment by ghogen** [comment]
@gewarren Yes, the feedback looks good. We're moving the source of truth for this content into the MSBuild repo, so rather than making the change here, I'll integrate the work into the appropriate place in the MSBuild repo.  I'll go ahead and close the PR here.

**Pull request closed without merging**